net/bind/Makefile

```diff
@@ -175,6 +175,7 @@ define Package/bind-tools/install
 	$(INSTALL_DIR) $(1)/usr/sbin
 	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/usr/sbin/dnssec-keygen $(1)/usr/sbin/
 	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/usr/sbin/dnssec-signzone $(1)/usr/sbin/
+	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/usr/sbin/dnssec-settime $(1)/usr/sbin/
```

Comment:
No, because the way this Makefile seems to be made, bind-tools doesn't reference to bind-dnssec etc, but applies all parts individually.